service/controller/orgpermissions/resource/membership/create.go

```diff
@@ -67,9 +75,29 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 	} else if err != nil {
 		return microerror.Mask(err)
+
+	} else if needsUpdate(orgReadClusterRoleBinding, existingClusterRoleBinding) {
```

Comment:
We check here if the subjects have changed and if they did we update the` clusterRoleBinding`. This was actually missing before.